google/cloud/security/common/data_access/csv_writer.py

```diff
@@ -172,13 +251,99 @@
     'entity',
     'email',
     'domain',
-    'bucket'
+    'bucket',
+]
+
+CLOUDSQL_INSTANCES_FIELDNAMES = [
+    'project_number',
+    'name',
+    'project',
+    'backendType',
```

Comment:
I realize that this may be a big change, but any chance that you could make the camelCase fieldnames snake_case, for consistency with the other fieldnames in this module? (I forgot whether mysql column names are case sensitive, I think sometimes they might be?)